x-pack/test/observability_ai_assistant_functional/tests/conversations/index.spec.ts

```diff
@@ -66,78 +85,84 @@ export default function ApiTest({ getService, getPageObjects }: FtrProviderConte
-                  function_call: {
-                    name: '',
-                    arguments: '',
-                    trigger: MessageRole.Assistant,
-                  },
-                  role: MessageRole.Assistant,
-                },
+            },
+            {
+              '@timestamp': '2024-04-18T14:29:01.876Z',
+              message: {
+                content:
+                  '{"screen_description":"The user is looking at http://localhost:5601/ftw/app/observabilityAIAssistant/conversations/new. The current time range is 2024-04-18T14:13:49.815Z - 2024-04-18T14:28:49.815Z.","learnings":[]}',
+                name: 'context',
+                role: MessageRole.User,
               },
-            ],
-            conversation: {
-              title: 'My old conversation',
-              token_count: {
-                completion: 1,
-                prompt: 1,
-                total: 2,
+            },
+            {
+              '@timestamp': '2024-04-18T14:29:22.945Z',
+              message: {
+                content:
+                  "SLOs, or Service Level Objectives, are a key part of the Site Reliability Engineering (SRE) methodology. They are a target value or range of values for a service level that is measured by an SLI (Service Level Indicator). \n\nAn SLO is a goal for how often and how much you want your service to meet a particular SLI. For example, you might have an SLO that your service should be up and running 99.9% of the time. \n\nSLOs are important because they set clear expectations for your team and your users about the level of service you aim to provide. They also help you make decisions about where to focus your efforts: if you're meeting your SLOs, you can focus on building new features; if you're not meeting your SLOs, you need to focus on improving reliability. \n\nIn Elastic Observability, you can define and monitor your SLOs to ensure your services are meeting their targets.",
+                function_call: {
+                  name: '',
+                  arguments: '',
+                  trigger: MessageRole.Assistant,
+                },
+                role: MessageRole.Assistant,
               },
             },
-            '@timestamp': '2024-04-18T14:29:22.948',
-            public: false,
-            numeric_labels: {},
-            labels: {},
+          ],
+          conversation: {
+            title: 'My old conversation',
+            token_count: {
+              completion: 1,
+              prompt: 1,
+              total: 2,
+            },
           },
+          '@timestamp': '2024-04-18T14:29:22.948',
+          public: false,
+          numeric_labels: {},
+          labels: {},
         },
       },
-    });
+    };
+    await supertestWithoutAuth
+      .post(endpoint)
+      .set('kbn-xsrf', 'xxx')
+      .set('Cookie', cookie)
+      .send(params.body);
```

Comment:
@arturoliduena Just noticed this while looking at @viduni94's [PR](https://github.com/elastic/kibana/pull/202817/). We should aim to use `observabilityAIAssistantAPIClient` when possible because it comes with type safety, making it easier to use, and "fails faster" when an endpoint is changed:

```ts
const fullConversation = await observabilityAIAssistantAPIClient.noAuth({
  endpoint: "GET /internal/observability_ai_assistant/conversation",
  params: { 
    // ... 
  },
});
```

`noAuth`  user doesn't exist but can be created [here](https://github.com/elastic/kibana/blob/01de8870602d032f0c49c791552966fa556e634e/x-pack/test/observability_ai_assistant_api_integration/common/users/users.ts#L36)